Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -0,0 +1,74 @@
+#import "SentryNetworkTracker.h"
+#import "SentryOptions+Private.h"
+#import "SentryPerformanceTracker.h"
+#import "SentrySDK+Private.h"
+#import <objc/runtime.h>
+
+static NSString *const SENTRY_NETWORK_REQUEST_TRACKER_SPAN_ID
+    = @"SENTRY_NETWORK_REQUEST_TRACKER_SPAN_ID";
+
+@interface
+SentryNetworkTracker ()
+
+@property (nonatomic, strong) SentryPerformanceTracker *tracker;
+
+@end
+
+@implementation SentryNetworkTracker
```

Comment:
We're specifically swizzling `NSURLSession` right? Would it make sense to be specific with the name here like `SentryURLSessionTaskTracker`?

Reply:
But we will swizzle NSURLConnection for sure after this.